src/css/custom.css

```diff
@@ -283,14 +283,14 @@ h1 {
 .user-survey {
     width: 100%;
     min-height: 600px;
+    height: 6175px;
```

Comment:
@DaanHoogland this might create a huge length between the survey block on homepage and footer. Probably you don't want to use it for homepage but only for the /survey page?